src/ontology/components/highlevelclass.tsv

```diff
@@ -1,47 +1,47 @@
\ No newline at end of file
+VBO:ID (start after the species ID)	label with species name	Parent ID	source for parents	source for parents	db-xref	Defintion	Source	Source	Equivalent	Equivalent	Comment	Synonym	Synonym Source	gh_issue_NOT_merge_obsolete		
+ID	LABEL	SC %	>A oboInOwl:source SPLIT=|	>AT oboInOwl:source^^xsd:anyURI SPLIT=|	A oboInOwl:hasDbXref	A IAO:0000115	>A oboInOwl:source SPLIT=|	>AT oboInOwl:source^^xsd:anyURI SPLIT=|	EC 'Vertebrate breed' and %	EC %	A rdfs:comment	A oboInOwl:hasExactSynonym	>AT oboInOwl:source^^xsd:anyURI SPLIT=|	AT IAO:0000233^^xsd:anyURI SPLIT=|		
+VBO:0000038	Alpaca breed	VBO:0400032	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153	https://www.fao.org/dad-is		A vertebrate breed that is a Vicugna pacos.	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153		NCBITaxon:30538		Breeds included in VBO have been characterized, defined, and determined by international breed organizations, communities, experts, and/or have been reported in the literature. https://monarch-initiative.github.io/vertebrate-breed-ontology/general/general/			https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/209		A vertebrate breed that is a Vicugna pacos.
+VBO:0000041	American bison breed	VBO:0400019	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153	https://www.fao.org/dad-is		A vertebrate breed that is a Bison bison.	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153		NCBITaxon:9901		Breeds included in VBO have been characterized, defined, and determined by international breed organizations, communities, experts, and/or have been reported in the literature. https://monarch-initiative.github.io/vertebrate-breed-ontology/general/general/			https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/209		Vertebrate breed: A vertebrate breed is a member of a group of vertebrate animals that share specific characteristics (such as traits, behavior, genetics) that distinguish it from other organisms of the same species, and/or for which cultural or geographical separation has led to the general acceptance of its separate identity. Breeds are formed through genetic isolation and either natural adaptation to the environment or selective breeding, or a combination of the two.
+VBO:0000068	Buffalo breed	VBO:0400019	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153	https://www.fao.org/dad-is	LBO:0001042	A vertebrate breed that is a Bubalus bubalis.	https://orcid.org/0000-0002-5002-8648|https://orcid.org/0000-0002-4142-7153		NCBITaxon:89462		Breeds included in VBO have been characterized, defined, and determined by international breed organizations, communities, experts, and/or have been reported in the literature. https://monarch-initiative.github.io/vertebrate-breed-ontology/general/general/			https://github.com/monarch-initiative/vertebrate-breed-ontology/issues/209		

```

Comment:
Remember that human readable definititions (as opposed to descriptions!) should be necessary and sufficient.

```
A vertebrate breed that is a Bubalus bubalis.
``` 

Makes limited sense to my eyes.

1. Either this breed _corresponds exactly to_ `Bubalus bubalis`, and then its not needed (you just use the NCBITaxon id?) or
2. There is more than one breed that  "is a Bubalus bubalis" and then the definition is insufficient, i.e. the differentiae is missing. 

If you tell me which it is I will help you with the right solution.

Reply:
Option2: there are 4 different breeds of Bubalus bubalis (American bison).

<img width="431" alt="Screenshot 2024-11-27 at 2 15 32 PM" src="https://github.com/user-attachments/assets/549ba6d9-2582-4c17-b0ea-a4bb3a4996eb">

Yes, please help me find the right solution :)

Reply:
Here is the key question:

What is the difference between "American bison breed" and https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=9901?

Reply:
I would have to say nothing. I think the rationale for adding the grouping classes, e.g. American bison breed is on potentially on shaky ground. These grouping terms (e.g. chicken breed, duck breed, cattle breed, dog breed, etc.) were added to make it easier to search for individual breeds in the hierarchy. 

Maybe in the case of cattle breed where the grouping class is at the level of Bos and therefore includes all members of Bos https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi, there is a difference between, for example bos taurus and "cattle breed," but this is not the case for grouping classes that are defined at the species or subspecies level.  

What do you recommend?

Reply:
If they are the same you will have to consult with @sabrinatoro and consider swapping in NCBITaxon ids.. How much effort would that be?

Reply:
These 2 taxons are different in NCBITaxon though:
"Bulbus" and "Bison" are 2 distinct groups of [Bovinae](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Undef&id=27592&lvl=3&lin=f&keep=1&srchmode=1&unlock)
Are we saying that the breeds should be children of both?

Reply:
If they are different then the answer to

> What is the difference between "American bison breed" and https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=9901?

Is not "nothing"; @katiermullen do I see this correctly that Sabrina believes they are indeed different?